src/lib/moderationList.ts

```diff
@@ -0,0 +1,21 @@
+import { readFileSync } from 'fs';
+import path from 'path';
+
+const FIELDS = ['flaggedLinks', 'flaggedProposalIds', 'verifiedSpaces'];
+const CACHE_PATH = path.resolve(__dirname, `../../${process.env.MODERATION_LIST_PATH || 'data'}`);
```

Comment:
Can default it to `data` no need for env :) or we can have it as a variable or config.json if required, should have less env as possible, in this case, I don't think we will ever change the folder name 😅

Reply:
`MODERATION_LIST_PATH` is used by the test suite, to avoid mocking

Reply:
maybe let's just use the relative path, i mean `data` folder inside `test` folder